docs/daemon/cli.mdx

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 2
+sidebar_label: "CLI"
+id: daemon-cmdline
+title: Kwild CLI Guide
+description: Kwild Command Line
+slug: /daemon/cmdline
+---
+
+Kwild command starts both the blockchain and the database services of kwild node. 
+
+### Quickstart a kwild node:
+Run the below command to quickstart a kwild node without specifying any configuration.
+```bash
+kwild --autogen  --root_dir "path/to/root/dir"
+```
+This command will generate the missing configuration in the root directory and starts the node with the generated configuration. 
+
+The default root directory used is `~/.kwild` and can be overwritten using `--root_dir`
+
+### Run node with specific configuration
+If root directory is already intialized with the required configuration, then node can be started just by running the below command:
+```bash
+kwild --root_dir "path/to/root/dir"
+```
+
+### Config override using flags
+The kwild command also supports overriding the configuration using the cmd line flags.  Use `--help or -h` for detailed information on all the flags supported by the kwild cli. For more information, refer [config override through flags](./configuration/configuration.mdx#config-override)
```

Comment:
I think we should use full names like "command line".  Once again on the proper grammar point, things like "For more information, refer to the [configuration guide](link).